src/main/java/pl/simpleascoding/tutoringplatform/api/publicResource/UserController.java

```diff
@@ -17,6 +20,7 @@
 class UserController {
```

Comment:
There should be validation annotations in methods

Reply:
Not related to this feature - will be fixed in a separate pull request